ReactiveXComponent/RabbitMq/RabbitMqSubscriber.cs

```diff
@@ -19,112 +19,128 @@ public class RabbitMqSubscriber : IXCSubscriber
         private readonly IXCConfiguration _xcConfiguration;
         private readonly string _privateCommunicationIdentifier;
 
-        private readonly Dictionary<SubscriberKey, RabbitMqSubscriberInfos> _subscribers;
-        private readonly Dictionary<SubscriberKey, List<Action<MessageEventArgs>>> _listenerToUpdateBySubscriberKeyRepo;
+        private readonly ConcurrentDictionary<SubscriberKey, RabbitMqSubscriberInfos> _subscribers;
+        private readonly ConcurrentDictionary<SubscriberKey, ThreadSafeList<Action<MessageEventArgs>>> _listenerToUpdateBySubscriberKeyRepo;
```

Comment:
You should avoid the _<object_name>To<Key_name>_ format when naming dictionaries. Here you ended up with a method called _AddToListenerToUpdateBySubscriberKeyRepository_ which is not readable.